packages/test/test-drivers/src/odspDriverApi.ts

```diff
@@ -39,6 +39,7 @@ const odspSnapshotOptions: OptionsMatrix<ISnapshotOptions> = {
     deltas: numberCases,
     mds: numberCases,
     timeout: numberCases,
+    cacheExpiryTimeoutOverrideMs: numberCases,
```

Comment:
This does not look entirely right. The value should be limited to the default / max timeout. Otherwise, the stress tests may use a timeout value higher resulting in an error being throw.